modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/FillMBLayer.java

```diff
@@ -146,7 +139,7 @@ public FillTranslateAnchor getFillTranslateAnchor() {
      * 4, 8, ..., 512).
      */
     public Expression getFillPattern() {
```

Comment:
The names of the Expression-returning methods are query methods and should leave off the "get" to conform to the convention we discussed for MBLayer.java